client/__tests__/e2e/e2e.test.js

```diff
@@ -522,7 +522,12 @@ test("lasso moves after pan", async () => {
   expect(panCount).toBe(initialCount);
 });
 
-describe("auth buttons", () => {
+const describeIfCalledByMakeFileTarget =
```

Comment:
Question - are you able to get the configuration values on the frontend? Then you could check whether the oauth `type==none` which would effectively be the same thing without needing to create the additional environment variable?

Reply:
I can make an API fetch from the e2e tests for the config endpoint.  How does that sound?

Reply:
SGTM 👍